package.json

```diff
@@ -147,7 +147,7 @@
     ],
     "moduleNameMapper": {
       "^@/(.*)$": "<rootDir>/src/$1",
-      "\\.(css)$": "identity-obj-proxy"
+      "^.+\\.(css|less|scss)$": "identity-obj-proxy"
```

Comment:
This sounds like we would be handing out `less` and `scss` files as they are.

Is that what you intended? I probably just don't understand what's going on here. :)

Reply:
Without this jest will complain as we have an import of an scss file in js being now pulled into the scripts loaded in the jest tests. Since we do not care about the styling there, we can just return a proxy instead. 

less was just added for completeness as this is a pattern i found while researching what to do :D